src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitReport;
+import core.basesyntax.service.ReportService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private ReportService reportService = new ReportServiceImpl();
+
+    @Before
+    public void cleanBefore() {
```

Comment:
Not fixed
![image](https://user-images.githubusercontent.com/117547096/214878953-1eb8646e-4f13-4a77-ba67-6453deec17f8.png)